output_results.go

```diff
@@ -11,19 +12,19 @@ func RunOutputTask(o io.Writer, e io.Writer, resourceFactory DumpedJSONResourceF
 	analysisResults := AnalysisResults{}
 	err := resourceFactory("analysis.json", &analysisResults)
 	if err != nil {
-		e.Write([]byte(err.Error()))
+		fmt.Fprintf(e, err.Error())
 	}
 
 	for scope, results := range analysisResults {
 		for _, result := range results {
 			if result.Status != 0 {
-				o.Write([]byte("Potential issue discovered in " + scope + ": " + result.CheckName + "\n"))
+				fmt.Fprintf(o, "Potential issue discovered in %s: %s\n", scope, result.CheckName)
 				o.Write([]byte("  Details:\n"))
 				for _, info := range result.Info {
-					o.Write([]byte("    " + strings.Replace(strings.TrimSpace(info.Message), "\n", "\n    ", -1) + "\n"))
+					fmt.Fprintf(o, "    %s\n", strings.Replace(strings.TrimSpace(info.Message), "\n", "\n    ", -1))
```

Comment:
Possibly overkill, but I knew godoc had code to rewrap and indent text:
https://play.golang.org/p/gRSnFC-gDp